remove -O3 and -march=native optimizations
fix man directory (linux-package puts man under share/man, ports expect man/)
disable SIMD on OpenBSD (simde not available as a port)
use gmake on OpenBSD for docs build
build only man pages on OpenBSD (furo theme and tab directives not available)
skip HTML docs installation on OpenBSD

Index: setup.py
--- setup.py.orig
+++ setup.py
@@ -625,7 +625,6 @@ def init_env(
         df += ' -Og'
         float_conversion = '-Wfloat-conversion'
     fortify_source = '' if sanitize and is_macos else '-D_FORTIFY_SOURCE=2'
-    optimize = df if debug or sanitize else '-O3'
     sanitize_args = get_sanitize_args(cc, ccver) if sanitize else []
     cppflags_ = os.environ.get(
         'OVERRIDE_CPPFLAGS',
@@ -647,12 +646,12 @@ def init_env(
         'OVERRIDE_CFLAGS',
         (
             f'-Wextra {float_conversion} -Wno-missing-field-initializers -Wall -Wstrict-prototypes {c_std}'
-            f' {werror} {optimize} {sanitize_flag} -fwrapv {stack_protector} {missing_braces}'
+            f' {werror} {sanitize_flag} -fwrapv {stack_protector} {missing_braces}'
             f' -pipe -fvisibility=hidden {no_plt}'
         ),
     )
     cflags = shlex.split(cflags_) + shlex.split(sysconfig.get_config_var('CCSHARED') or '')
-    ldflags_ = os.environ.get('OVERRIDE_LDFLAGS', '-Wall ' + ' '.join(sanitize_args) + ('' if debug else ' -O3'))
+    ldflags_ = os.environ.get('OVERRIDE_LDFLAGS', '-Wall ' + ' '.join(sanitize_args))
     ldflags = shlex.split(ldflags_)
     ldflags.append('-shared')
     cppflags += env_cppflags
@@ -725,9 +724,6 @@ def init_env(
     if control_flow_protection:
         cflags.append(control_flow_protection)
 
-    if native_optimizations and ba.isa in (ISA.AMD64, ISA.X86):
-        cflags.extend('-march=native -mtune=native'.split())
-
     ans = Env(
         cc,
         cppflags,
@@ -1269,7 +1265,7 @@ def build_wayland_protocols(
 
 
 def compile_glfw(compilation_database: CompilationDatabase, build_dsym: bool = False) -> None:
-    modules = 'cocoa' if is_macos else 'x11 wayland'
+    modules = 'cocoa' if is_macos else ('x11' if is_openbsd else 'x11 wayland')
     for module in modules.split():
         try:
             genv = glfw_init_env(env, pkg_config, pkg_version, at_least_version, test_compile, module)
@@ -1400,7 +1396,7 @@ def build_shaders(args: Options, kitty_exe: str, for_f
         return
     env = os.environ.copy()
     env['ASAN_OPTIONS'] = 'detect_leaks=0'
-    cp = subprocess.run(
+    cp = subprocess.CompletedProcess([], 0) if is_openbsd else subprocess.run(
         [
             kitty_exe,
             '+launch',
@@ -1611,8 +1607,6 @@ def build_launcher(args: Options, launcher_dir: str = 
             cflags.extend(sanitize_args)
             ldflags.extend(sanitize_args)
             libs += ['-lasan'] if not is_macos and env.compiler_type is not CompilerType.clang else []
-    else:
-        cflags.append('-g3' if args.debug else '-O3')
     if bundle_type.endswith('-freeze'):
         cppflags.append('-DFOR_BUNDLE')
         cppflags.append(f'-DPYVER="{sysconfig.get_python_version()}"')
@@ -1678,7 +1672,7 @@ def build_launcher(args: Options, launcher_dir: str = 
 
 # Packaging {{{
 def copy_man_pages(ddir: str) -> None:
-    mandir = os.path.join(ddir, 'share', 'man')
+    mandir = os.path.join(ddir, 'man')
     safe_makedirs(mandir)
     man_levels = '15'
     with suppress(FileNotFoundError):
@@ -1744,8 +1738,8 @@ def create_linux_bundle_gunk(ddir: str, args: Options)
         kitten_exe = os.path.join(os.path.dirname(str(launcher)), 'kitten')
         if os.path.exists(kitten_exe):
             os.environ['KITTEN_EXE_FOR_DOCS'] = kitten_exe
-            make = 'gmake' if is_freebsd else 'make'
-            run_tool([make, 'docs'])
+            make = 'gmake' if is_freebsd or is_openbsd else 'make'
+            run_tool([make, 'man' if is_openbsd else 'docs'])
         else:
             if args.skip_building_kitten:
                 skip_docs = True
@@ -1759,7 +1753,8 @@ def create_linux_bundle_gunk(ddir: str, args: Options)
                 raise SystemExit(f'kitten binary not found at: {kitten_exe}')
     if not skip_docs:
         copy_man_pages(ddir)
-        copy_html_docs(ddir)
+        if not is_openbsd:
+            copy_html_docs(ddir)
     for icdir, ext in {'256x256': 'png', 'scalable': 'svg'}.items():
         icdir = os.path.join(ddir, 'share', 'icons', 'hicolor', icdir, 'apps')
         safe_makedirs(icdir)
